repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -125,3 +129,8 @@ object ProjectUpdateFileRequest {
       unlift(ProjectUpdateFileRequest.unapply)
     )
 }
+
+final case class Message(text: String)
```

Comment:
**My expectations about it:**
This, I suppose, will be replaced later by classes with more concrete replies. Now it describes common cases: For example, it needs for cases where GitLab just replies to us with a notification that it created file or not, etc
like:
{
 'smth':'project created'
}

For responses which are not bear some informative work, have any informative value.(we have a response status, and we know that it succeed). Or for situations where information in response bodies will be evaluated later...

So the better name for it may be GitLabResponseMessage ?

**And "pragmatic" answer:**
The point is that @GrigoriiBerezin has some "mock" answers in his testcases set, and I have parametrized client, so I had to encapsulate String response bodies into types.

I have  ADT:
```
final case class SuccessResponseBody[B](bodyObject: B) extends ResponseBody[B]
final case class FailureResponseBody[B](error: String) extends ResponseBody[B]
```
but sometimes we get in response not informative body, but some answer. So, this Message type is for that.

Reply:
Thank you for such a detailed answer, but it does not contain the answer to my question. 
The presence of field `text` in this class requires that field with such name MUST exist in response. Are you sure that this field exists in all responses where you use this`Message`?
You correctly pointed out that this is kinda mock for a few cases where you don't care about response body. I think it must be mentioned in the class name, rather than the fact that this is a message of the success response.

Reply:
1) no, field "text" is just my "interpretation". I have no sense about "text" field in real GitLab response
2) Does the name "SuccessResponseMessage" fit, in your opinion?

Reply:
1) In that case, you should not use a class with such a structure.
2) In my opinion, it doesn't fit. There is no way that we can understand that this is a mock object for the response rather than a real one. For example, names like `DummyResponse`, `MockResponse`, `EmptyResponse`  may give a hint to another developer that something is wrong with such responses and that he should not use it whenever he wants.
If you think that in your cases the response is just a string, that you should do something like that
`post[String, OutputType](???)`
`json` that consists of one string is **not** the same thing as `jsob` object that contains a single string field.

Reply:
I think you did not understand me. This class is just a wrap for responseBody. It is not a mock. While I have parametrized Response and parametrized ResponseBody I have a new duty - to pack my success response body into any kind of "wrap". 

```
case class Response[B](status: Int, body: ResponseBody[B], headers: Map[String, Seq[String]])

sealed trait ResponseBody[B]

final case class SuccessResponseBody[B](bodyObject: B) extends ResponseBody[B]
```
The "Message" is just a wrap, maybe the best name for it will be - "SuccessResponseBodyContent" - its too long, but it clear what it does